src/client.js

```diff
@@ -16,13 +19,15 @@ export class Saturn {
    * @param {string} [opts.cdnURL=saturn.ms]
    * @param {number} [opts.connectTimeout=5000]
    * @param {number} [opts.downloadTimeout=0]
+   * @param {string} [opts.orchURL]
    * @param {import('./storage/index.js').Storage} [opts.storage]
    */
   constructor (opts = {}) {
     this.opts = Object.assign({}, {
       clientId: randomUUID(),
       cdnURL: 'saturn.ms',
       logURL: 'https://twb3qukm2i654i3tnvx36char40aymqq.lambda-url.us-west-2.on.aws/',
+      orchURL: 'https://orchestrator.strn.pl/nodes?maxNodes=100',
```

Comment:
@guanzo I want to clean this up a little by having those URLs in one place and they get imported here. You cool with that?

Reply:
Do we want to keep our auth URL at least a private var? 

Doesn't completely hide it but at least makes it harder for someone to find it and use it.

Reply:
These variables are usually env vars injected during the build process. 

When you guys say "private var" do you mean https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields?

Reply:
yes thats correct.